### PR TITLE
Update list.txt

### PR DESCRIPTION
**Pull request by JoseVm96**

### CODE REVIEW

File: Attendance/list.txt [hunk 1]
```diff
@@ -1,4 +1,3 @@
 Sebastian Herrera Bermeo |25|
 Erick Eduardo Hernández Rodríguez |25|
-Vargas Martínez José de Jesús |25|
-Vargas Martínez José de Jesús |26|
+Vargas Martínez José de Jesús |25||26||27|
```
Comment (luisalanmz) [inline Attendance/list.txt:3]: Try to put just one Pipe

**Review by luisalanmz — CHANGES_REQUESTED** [review]
Just one suggestion

File: Attendance/list.txt (hunk 1 above)
Comment (JoseVm96) [reply Attendance/list.txt:3]: I’ll modify it in my next attendance.

Comment (JoseVm96) [reply Attendance/list.txt:3]: Thank you!

**Review by luisalanmz — APPROVED** [review]
OK

**Review by globant-bootcamp — APPROVED** [review]

**Pull request closed without merging**